.github/workflows/run_tests.yml

```diff
@@ -20,6 +20,9 @@ jobs:
         with:
           python-version: 3.9
 
+      - name: Build service image
+        run: ./bin/build-image
+
```

Comment:
I was trying to figure out exactly what was going on, but nothing in the PR publishes a new image to ghcr.io.

Reply:
This change primarily is to accommodate the changes to `tests.Dockerfile` (again to align that image to what's happening with other services). The `tests.Dockerfile` used to replicate all the service image building that is done in `service.Dockerfile`, whereas it now uses the service Docker image as a base image, meaning that service image also needs to be built. (I'm probably just repeating stuff you worked out, sorry if that's the case)

It's interesting with the workflows (you have a comment elsewhere about not seeing old runs) - I am 90% sure they ran previously, but it's been a while since this repository last had a merge into `main` (April 2023). I know artefacts eventually get removed, I wonder if particularly old workflow runs also get cleaned up, too?

Reply:
I did not work that out, and appreciate the update.